src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/ContaoBackendViewTemplate.php

```diff
@@ -13,15 +13,58 @@
 namespace ContaoCommunityAlliance\DcGeneral\Contao\View\Contao2BackendView;
 
 use ContaoCommunityAlliance\DcGeneral\View\ViewTemplateInterface;
+use ContaoCommunityAlliance\Translator\TranslatorInterface;
 
 /**
  * This class is used for the contao backend view as template.
  *
  * @package DcGeneral\View
  */
-class ContaoBackendViewTemplate extends \BackendTemplate implements ViewTemplateInterface
+class ContaoBackendViewTemplate extends \BackendTemplate implements ViewTemplateInterface, TranslatorInterface
 {
     /**
+     * The translator.
+     *
+     * @var TranslatorInterface
+     */
+    protected $translator;
+
+    /**
+     * Construct.
+     *
+     * @param string              $strTemplate    The template name.
+     * @param string              $strContentType the content type.
+     */
+    public function __construct($strTemplate = '', $strContentType = 'text/html')
+    {
+        parent::__construct($strTemplate, $strContentType);
+    }
+
+    /**
+     * Get the translator.
+     *
+     * @return TranslatorInterface
+     */
+    public function getTranslator()
+    {
+        return $this->translator;
+    }
+
+    /**
+     * Set the translator.
+     *
+     * @param TranslatorInterface $translator The translator.
+     *
+     * @return $this
+     */
+    public function setTranslator(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+
+        return $this;
+    }
```

Comment:
Yep, we should not change the constructor.
It would work for most implementations when we let the argument default to `null` but as we have to use the setters on the template anyway we can also call this method then. The `BaseView` already has a `getTemplate` method which can issue this call inline.